src/Filters/GravityFormsStyles.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Sets the default styling for Gravity Forms
+ *
+ * @since 1.4.1
+ *
+ * @package Bcgov/Theme/Block
+ */
+
+namespace Bcgov\Theme\Block\Filters;
+
+/**
+ * This class allows you to set the default style options for Gravity Forms. By default, these are set to use values that
+ * are consistent with BC Gov styling. These values can be changed as needed.
+ *
+ * @since 1.4.1
+ */
+class GravityFormsStyles {
+
+	/**
+	 * Set default style values for Gravity Forms plugin
+	 *
+	 * @param  string $styles Current styles for the Gravity Form.
+	 * @return string
+	 */
+	public function add_gravity_forms_default_styles( $styles ) {
```

Comment:
should add a reference to point to the design system, where these colours are defined